components/pagination/Pagination.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * Pagination.jsx
+ * Created by Lizzie Salita 10/16/17
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { formatNumberWithPrecision } from '../../helpers/moneyFormatter';
+import { calculatePageRange } from '../../helpers/paginationHelper';
+
+import Pager from './Pager';
+import LimitSelector from './LimitSelector';
+import GoToPage from './GoToPage';
+
+const propTypes = {
+    changePage: PropTypes.func.isRequired,
+    totalItems: PropTypes.number.isRequired,
+    currentPage: PropTypes.number,
+    pageSize: PropTypes.number,
+    resultsText: PropTypes.bool,
+    customResultsText: PropTypes.element,
```

Comment:
Shouldn't this be a string?

Reply:
[Here's an example](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/components/award/idv/activity/IdvActivityVisualization.jsx#L165) of where we use more markup than just a string, so I made it an element to allow for more flexibility.